guardian/recipes
# Tidying and refactoring

## What does this change?

A bit of housekeeping, this PR refactors some of the new code to reduce duplication, remove errors, and hopefully make the whole code base that little bit more manageable. 

There are no changes to Hatch functionality, this is all under the hood.

## How to test

Run locally, use the affected pages and components to check things are working as normal.

## How can we measure success?

Slightly tidier, easier to use codebase.
